app/helpers/articles_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ArticlesHelper
+end
```

Comment:
can get rid of this file. it's auto generated but only used if you add helper methods in it (which you haven't (which is likely good))